crates/p2p/src/events.rs

```diff
@@ -1,12 +1,30 @@
 //! Events emitted from P2P.
 
 /// Events emitted from P2P that needs to be handled by the user.
+#[cfg(feature = "request-response")]
+use tokio::sync::oneshot::Sender;
+
+/// Events emitted from the gossipsub protocol.
 
```

Comment:
remove this empty line.
Don't know how `cargo doc` didn't error with this "standalone" docstring...